### PR TITLE
Standardize .travis.yml, fix line endings

### PR DESCRIPTION
**Pull request by meresmclr**
* Using standard `.travis.yml` syntax, successful tests in Python 2.7,3.4,3.5.
* Fixed line endings from CRLF to standard LF

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,55 +1,62 @@
...
-  - conda update --yes conda
-  
-  # system libs
-  - sudo apt-get install libraw-dev liblensfun-dev libnetcdf-dev libhdf5-serial-dev libimage-exiftool-perl gfortran
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH=$HOME/miniconda/bin:$PATH
+  - hash -r
   
+  # CDF
+  - wget -r -l1 -np -nd -nc http://cdaweb.gsfc.nasa.gov/pub/software/cdf/dist/latest-release/linux/ -A cdf*-dist-all.tar.gz
```
Comment (letmaik) [inline .travis.yml]: Can you enlighten me what all the wget parameters are doing?

File: .travis.yml [hunk 2]
```diff
@@ -1,55 +1,62 @@
+sudo: required
+
 language: python
 
+dist: trusty
+
+notifications:
+    email: false
```
Comment (letmaik) [inline .travis.yml]: I think we do want notifications. Any specific reason you think this is not a good idea?

Comment (meresmclr) [reply .travis.yml]: Oh yes since there's normally not an error now

**Comment by letmaik** [comment]
In turn with fixing the line endings it's probably a good idea to add a .gitattributes file which prevents future mistakes, right?

File: .travis.yml (hunk 1 above)
Comment (meresmclr) [reply .travis.yml]: actually `-nc` shouldn't be there, that says don't overwrite. Travis always has a fresh image for free tier, and in any case we would want to clobber with a new version if one existed.
Those options are to allow automatically getting the latest version without specifying version number.

-r recursive
-l1 down one level only
-np don't traverse up a directory
-nd don't save directory hierarchy

**Comment by meresmclr** [comment]
I have applied all your changes.

**Pull request merged**